sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseFacade.java

```diff
@@ -205,7 +205,7 @@ void saveBulkEditWithFacilities(
 
 	List<CasePersonDto> getDuplicates(@Valid CasePersonDto casePerson);
 
-	List<CaseDataDto> getDuplicatesWithPathogenTest(@Valid CaseDataDto caseDataDto, PathogenTestDto pathogenTestDto);
+	List<CaseDataDto> getDuplicatesWithPathogenTest(@Valid String casePersonUuid, PathogenTestDto pathogenTestDto);
```

Comment:
To make this method's signature more concise, do not use parameter type `String casePersonUuid`, but `PersonReferenceDto personReference` instead.
This avoids calling the method with an arbitrary `String` and also avoids extracting the person's UUID before calling the method in various places.